### DISCUSSION TITLE
Can't get any Shipping method on checkout.

### DISCUSSION THREAD

**Discussion by ynlevi** [swellstores/community#549] [Help]
I am getting "There are no shipping services available for your location", after filling the address, although all the shipping should fit with the setting I've set on setting>shipping

I also looked at the cart api and I get empty string on shipmentRating.services

Thank you very mush for your help

<img width="1206" alt="Screenshot 2024-06-22 at 14 48 02" src="https://github.com/swellstores/community/assets/107810197/e6847fc1-c11b-46b2-90f0-34b5031ed885">
<img width="1189" alt="Screenshot 2024-06-22 at 15 40 27" src="https://github.com/swellstores/community/assets/107810197/1967a672-c566-4dec-9c4e-f1ae8eed053d">
<img width="821" alt="Screenshot 2024-06-22 at 21 34 26" src="https://github.com/swellstores/community/assets/107810197/ad72a45a-cc9b-4cc2-b130-82946dda8480">

**Comment by sags95** [comment]
Have you changed the store's main currency from USD?

**Reply by ynlevi** [reply]
Hi!
Yes I did🤗

**Reply by sags95** [reply]
Would you be able to change it again, then back to whichever currency you were using?
